### PR TITLE
web: Switch to new REST object APIs

### LINKED ISSUE #214: Switch to new REST object APIs
## Is your feature request related to a problem? Please describe.
I'm always frustrated when we're not using our new shiny REST APIs. 0.9.0 finally has some good ones for objects.

## Describe the solution you'd like
Switch object upload/download to new APIs.

## Describe alternatives you've considered
Zero alternatives.

## Additional context
https://github.com/nspcc-dev/neofs-rest-gw/releases/tag/v0.9.0

### PR DESCRIPTION
**Pull request by mike-petrov**
closes #214

Signed-off-by: Mikhail Petrov <mike@nspcc.ru>

### CODE REVIEW

**Comment by mike-petrov** [comment]
@tatiana-nspcc @roman-khimov  Should we use `?fullBearer=true` or it's or is this gonna be changed and outdated?

**Comment by roman-khimov** [comment]
Depends on https://github.com/nspcc-dev/neofs-rest-gw/pull/236. I think it can be avoided.

**Comment by mike-petrov** [comment]
`?fullBearer=true` removed

File: src/App.js [hunk 1]
```diff
@@ -47,14 +47,6 @@ export const App = () => {
 		scriptHash: Neon.create.account(process.env.REACT_APP_NEOFS_ACCOUNT).scriptHash,
 	});
 
-	const [ContentTypeHeader] = useState("Content-Type");
-	const [AuthorizationHeader] = useState("Authorization");
-	const [BearerOwnerIdHeader] = useState("X-Bearer-Owner-Id");
-	const [BearerForAllUsers] = useState("X-Bearer-For-All-Users");
-	const [BearerSignatureHeader] = useState("X-Bearer-Signature");
-	const [BearerSignatureKeyHeader] = useState("X-Bearer-Signature-Key");
-	const [BearerLifetime] = useState("X-Bearer-Lifetime");
```
Comment (roman-khimov) [inline src/App.js:65]: Not sure why you're removing these constants.

Comment (mike-petrov) [reply src/App.js:65]: I don't think it's best practice to do that, I've never seen it. Also, now we have less queries and there is no need to store these constants

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**